api/v1alpha3/machine_types.go

```diff
@@ -89,6 +89,11 @@ type MachineSpec struct {
 	// Must match a key in the FailureDomains map stored on the cluster object.
 	// +optional
 	FailureDomain *string `json:"failureDomain,omitempty"`
+
+	// NodeDrainTimeout is the total amount of time for draining a worker node
+	// Note that this NodeDrainTimeout is different from `kubectl drain --timeout`
```

Comment:
```suggestion
	//
	// NOTE: NodeDrainTimeout is different from `kubectl drain --timeout`
```

Let's also add some color on how it's different?

Reply:
Should we have a default here?